apps/ledger-live-desktop/src/newArch/features/Collectibles/Ordinals/components/Loader.tsx

```diff
@@ -0,0 +1,18 @@
+import { Flex, InfiniteLoader } from "@ledgerhq/react-ui";
+import React from "react";
+
+type Props = {
+  isLoading: boolean;
+};
+
+const Loader: React.FC<Props> = ({ isLoading }) => {
+  if (!isLoading) return null;
+
+  return (
+    <Flex justifyContent="center" my={4}>
+      <InfiniteLoader />
+    </Flex>
+  );
+};
+
+export default Loader;
```

Comment:
You should avoid do this `if !isLoading`

It's better to handle that on the component that this is used